app/src/main/java/org/simple/clinic/drugs/selectionv2/PrescribedDrugsScreenControllerV2.kt

```diff
@@ -35,7 +35,8 @@ class PrescribedDrugsScreenControllerV2 @Inject constructor(
         handleDoneClicks(replayedEvents),
         populateDrugsList(replayedEvents),
         addNewPrescription(replayedEvents),
-        selectDosage(replayedEvents))
+        selectDosage(replayedEvents),
+        updateCustomPrescription(replayedEvents))
```

Comment:
This function name suggests that it is going to also update, which is not true. Let's rename this to `showUpdateCustomPrescription()` or something?

Reply:
This name is along the same lines as `addNewPrescription` in this file.